Podfile.lock

```diff
@@ -16,4 +16,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 1985213b54ee7ae4e9f82fc14acc408fd9e87934
 
-COCOAPODS: 1.1.0.rc.2
+COCOAPODS: 1.0.1
```

Comment:
Please run pod install with 1.1.0.rc.2

This downgrades the version and maybe makes changes you did not intend.